core/templates/pages/exploration-editor-page/services/user-email-preferences-backend-api.service.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { RequestParams } from './user-email-preferences.service';
+
+/**
+ * @fileoverview User exploration emails backend api service
+ * for the exploration settings.
+ */
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserEmailPreferencesBackendApiService {
+  constructor(
+    private http: HttpClient,
+  ) { }
+
+  saveChangeToBackend(
+      requestParams: RequestParams
+  ): Promise<void | object> {
+    var emailPreferencesUrl = '/createhandler/notificationpreferences/12345';
```

Comment:
This seems incorrect. What does 12345 represent?

Reply:
Yes, that was a mistake. Fixed it now.